src/controllers/resource-controller.js

```diff
@@ -0,0 +1,105 @@
+const {responseServer, responseGeneral, responseNotFound} = require('../helpers/response-result');
+const {StatusCodes} = require ('http-status-codes');
+const Accounts = require('../models/accounts');
+const Lessons = require('../models/lessons');
+const Questions = require('../models/questions');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8292860)
```suggestion
const Questions = require("../models/questions");
```